### PR TITLE
Summary: rendered sponsor cards

### PR DESCRIPTION
**Pull request by david30907d**
## Types of changes

* **New feature**

## Description

need these components to display sponsorship

* my:
Screen Shot 2022-04-09 at 9 27 55 PM

* figma: 👇  https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=0%3A1
Screen Shot 2022-04-09 at 9 25 25 PM

## Steps to Test This Pull Request

1. `npm run dev`
2. navigate to <http://localhost:3000/2021/zh-hant>
3. scroll down and u'll see these sponsorship cards

## Expected behavior

* figma: 👇  https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=0%3A1
Screen Shot 2022-04-09 at 9 25 25 PM

## Related Issue

N/A

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -74,20 +75,26 @@
...
     </div>
 </template>
 
 <script>
 import { mapState } from 'vuex'
 import i18n from '@/i18n/index.i18n'
 import TextButton from '~/components/core/buttons/TextButton'
+import SponsorCard from '~/components/sponsors/SponsorCard'
+import SponsorModal from '~/components/sponsors/SponsorModal'
+import SponsorCardCollection from '~/components/sponsors/SponsorCardCollection'
```
Comment (david30907d) [inline pages/index.vue:88]: import these existing `SponsorCard` for rendering sponsor pages

File: pages/index.vue [hunk 2]
```diff
@@ -74,20 +75,26 @@
                 <text-button to="/sponsor">{{ $t('sponsorUs') }}</text-button>
             </div>
         </i18n-page-wrapper>
-        <sponsor-modal v-model="isOpened" :context="selectedSponsor" />-->
+        <sponsor-modal v-model="isOpened" :context="selectedSponsor" />
```
Comment (david30907d) [inline pages/index.vue:78]: actually I just uncommented the existing sponsorship cards (plz check this [commit](https://github.com/pycontw/pycontw-frontend/blob/4f854d7f92c5dc4e3192d00c2ab373688dbf1916/pages/index.vue#L49-L77) for details)

File: components/sponsors/SponsorCard.vue [hunk 3]
```diff
@@ -36,9 +36,9 @@ export default {
 <style lang="postcss" scoped>
 .sponsorCard {
     @apply relative flex flex-col justify-center items-center bg-white rounded-2xl;
-    width: 40%;
+    width: 25%;
     @media (min-width: 767px) {
-        @apply w-40;
+        @apply w-1/5;
```
Comment (david30907d) [inline components/sponsors/SponsorCard.vue:41]: I want 4 sponsorship cards in a row, but have no idea why we need to specify `w-1/5` instead of `w-1/4`

**Comment by david30907d** [comment]
@mattwang44

**Comment by mattwang44** [comment]
This PR will not be reviewed/merged until the styling part is finished. @josix will handle the rest in the following week 🤞🏽

**Comment by josix** [comment]
Updating at PR#225

**Pull request closed without merging**